db/migrate/20220425102037_create_users.rb

```diff
@@ -4,7 +4,6 @@ def change
       t.string :name
       t.string :photo
       t.text :bio
-
```

Comment:
- [ ] Your migrations are not working, as the previous reviewer stated.  Look:

![image](https://user-images.githubusercontent.com/57421823/166241642-bd14c316-72a2-40e4-a2ac-689728289498.png)

Maybe this is working for you, because you have your schema.rb already generated, but not for anyone else.

In order to debug this, you will need to run:

```bash
rake db:migrate VERSION=0
```

This will revert all your migrations to the database.   Then, try to run de migrations again that way you will be able to see the error and debug them.  Every time you see the migrations do not work, run this command again to revert them and try to debug again.

Please, make sure your migrations are free of errors.